curriculum/challenges/english/25-front-end-development/quiz-javascript-variables-and-data-types/66edc25ae5ea80bf6f785552.md

```diff
@@ -428,79 +418,65 @@ Which of the following is NOT a valid string concatenation method in JavaScript?
 
 #### --text--
 
-What will be the output of the following code:
```

Comment:
Maybe a question about which variable name is a valid identifier?

Reply:
good idea, those questions are always good